frontend/src/dataProvider.ts

```diff
@@ -38,6 +38,23 @@ export const dataProvider = {
         const { json } = await httpClient(url, params);
         return { data: json };
     },
+    getManyReference: async (resource, params) => {
+        const { page, perPage } = params.pagination;
+        const { field, order } = params.sort;
+        const queryparams = {
+            [params.target]: params.id,
+            ...params.pagination, 
+            ...params.sort, 
+            ...params.filter,
+        };
+
+        const url = `${apiUrl}/api/${resource}?${stringify(queryparams)}`;
+        const { json, headers } = await httpClient(url);
+        return {
```

Comment:
Note: the document body will only be shown (1) on a get request and (2) after ingestion.

Reply:
It's fine - if there's no body it will just be empty. We could show a snippet in the list view, I guess